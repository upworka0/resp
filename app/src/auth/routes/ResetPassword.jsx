import React from 'react';
import {
  Form, FormGroup, Label, Input,
  Button, Card, Row, Col, CardBody,
  CardHeader, FormFeedback,
} from 'reactstrap';

export default function ResetPassword() {
  return (
    <section className="d-flex flex-grow-1 flex-column align-items-center justify-content-center">
      <div className="container">
        <Row noGutters>
          <Col
            md={{
              size: 8,
              offset: 2,
            }}
            lg={{
              size: 6,
              offset: 3,
            }}
          >
            <h1 className="text-center mb-3">Loji</h1>
            <Card className="border-0 shadow-z4">
              <CardHeader className="bg-primary p-1" />
              <CardBody>
                <Form>
                  <FormGroup>
                    <Label htmlFor="loginInput">
                      New password
                    </Label>
                    <Input
                      type="email"
                      className="form-control"
                      id="loginInput"
                      placeholder="enter your email address"
                    />
                    <FormFeedback>Field is required</FormFeedback>
                  </FormGroup>
                  <FormGroup>
                    <Label htmlFor="loginInput">
                      Confirm new password
                    </Label>
                    <Input
                      type="email"
                      className="form-control"
                      id="loginInput"
                      placeholder="enter your email address"
                    />
                    <FormFeedback>Field is required</FormFeedback>
                  </FormGroup>
                  <div className="d-flex justify-content-center">
                    <Button type="submit" color="primary">Change Password</Button>
                  </div>
                </Form>
              </CardBody>
            </Card>
          </Col>
        </Row>
      </div>
    </section>
  );
}
