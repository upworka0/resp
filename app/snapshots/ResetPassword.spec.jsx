import React from 'react';
import { shallow } from 'enzyme';

import ResetPassword from '../src/auth/routes/ResetPassword';

describe('Reset Password UI', () => {
  it('should render reset password form', () => {
    const wrapper = shallow(<ResetPassword />);
    expect(wrapper).toMatchSnapshot();
  });
});
