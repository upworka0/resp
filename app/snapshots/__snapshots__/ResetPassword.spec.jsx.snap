// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reset Password UI should render reset password form 1`] = `
<section
  className="d-flex flex-grow-1 flex-column align-items-center justify-content-center"
>
  <div
    className="container"
  >
    <Row
      noGutters={true}
      tag="div"
    >
      <Col
        lg={
          Object {
            "offset": 3,
            "size": 6,
          }
        }
        md={
          Object {
            "offset": 2,
            "size": 8,
          }
        }
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <h1
          className="text-center mb-3"
        >
          Loji
        </h1>
        <Card
          className="border-0 shadow-z4"
          tag="div"
        >
          <CardHeader
            className="bg-primary p-1"
            tag="div"
          />
          <CardBody
            tag="div"
          >
            <Form
              tag="form"
            >
              <FormGroup
                tag="div"
              >
                <Label
                  htmlFor="loginInput"
                  tag="label"
                  widths={
                    Array [
                      "xs",
                      "sm",
                      "md",
                      "lg",
                      "xl",
                    ]
                  }
                >
                  New password
                </Label>
                <Input
                  className="form-control"
                  id="loginInput"
                  placeholder="enter your email address"
                  type="email"
                />
                <FormFeedback
                  tag="div"
                >
                  Field is required
                </FormFeedback>
              </FormGroup>
              <FormGroup
                tag="div"
              >
                <Label
                  htmlFor="loginInput"
                  tag="label"
                  widths={
                    Array [
                      "xs",
                      "sm",
                      "md",
                      "lg",
                      "xl",
                    ]
                  }
                >
                  Confirm new password
                </Label>
                <Input
                  className="form-control"
                  id="loginInput"
                  placeholder="enter your email address"
                  type="email"
                />
                <FormFeedback
                  tag="div"
                >
                  Field is required
                </FormFeedback>
              </FormGroup>
              <div
                className="d-flex justify-content-center"
              >
                <Button
                  color="primary"
                  tag="button"
                  type="submit"
                >
                  Change Password
                </Button>
              </div>
            </Form>
          </CardBody>
        </Card>
      </Col>
    </Row>
  </div>
</section>
`;
